_api-reference/profile.md

```diff
@@ -0,0 +1,754 @@
+                    "reason": "search_count",
+                    "time_in_nanos": 78626
+                  },
+                  {
+                    "name": "ProfilingAggregator: [avg_taxful_total_price]",
+                    "reason": "aggregation",
+                    "time_in_nanos": 2834566
+                  }
+                ]
+              }
+            ]
+          }
+        ],
+        "aggregations": [
+          {
+            "type": "AvgAggregator",
+            "description": "avg_taxful_total_price",
+            "time_in_nanos": 1973702,
+            "breakdown": {
+              "reduce": 0,
+              "post_collection_count": 1,
+              "build_leaf_collector": 199292,
+              "build_aggregation": 13584,
+              "build_aggregation_count": 1,
+              "build_leaf_collector_count": 2,
+              "post_collection": 6125,
+              "initialize": 6916,
+              "initialize_count": 1,
+              "reduce_count": 0,
+              "collect": 1747785,
+              "collect_count": 4675
+            }
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+</details>
+
+### Response fields
+
+The `aggregations` array contains aggregation objects with the following fields.
+
+Field | Data type | Description
+:--- | :--- | :---
+`type` | String | The aggregator type. In the [non-global aggregation example response](#example-response-non-global-aggregation), the aggregator type is `AvgAggregator`. [Global aggregation example response](#example-request-global-aggregation) contains a `GlobalAggregator` with a `AvgAggregator` child.
+`description` | String | Contains Lucene explanation of the aggregation. Helps differentiate aggregations with the same type.
+`time_in_nanos` | Long | The time the aggregation took to execute, in nanoseconds. In a parent aggregation, the time is inclusive of execution times of all the child aggregations.
+[`breakdown`](#the-breakdown-object-1) | Object | Contains timing statistics about low-level Lucene execution.
+`children` | Array of objects | If an aggregation has subaggregations (children), this field contains information about the subaggregations.
+`debug` | Object | Some aggregations return a `debug` object that describes details of the underlying execution.
+
+### The `breakdown` object
+
+The `breakdown` object represents the timing statistics about low-level Lucene execution, broken down by method. Each field in the `breakdown` object represents an internal Lucene method executed within the aggregation. Timings are listed in wall-clock nanoseconds and are not normalized. The `breakdown` timings are inclusive of all children times. The `breakdown` object is comprised of the following fields. All fields contain integer values.
+
+Field | Description
+:--- | :--- 
+`initialize` | Contains the time for the execution of `preCollection()` callback method during `AggregationCollectorManager` creation.
```

Comment:
Either "Contains the amount of time required to execute the" or "Contains the amount of time taken to execute the".